src/transaction.js

```diff
@@ -986,6 +1022,9 @@ Transaction.prototype.__toBuffer = function (buffer, initialOffset, __allowWitne
     var mask = (this.overwintered ? 1 : 0)
     writeInt32(this.version | (mask << 31))  // Set overwinter bit
     writeUInt32(this.versionGroupId)
+  } else if (this.isDashSpecialTransaction()) {
+    writeInt16(this.version)
+    writeInt16(this.type)
```

Comment:
It seems that these two writes should be `writeUInt16` per [DIP002](https://github.com/dashpay/dips/blob/master/dip-0002.md#special-transactions). This is already merged but I have another PR coming shortly to add special transaction signing to this library. I will include this fix there.

Reply:
See #40